api/Blob.json

```diff
@@ -209,7 +209,7 @@
               }
             ],
             "chrome_android": {
-              "version_added": true
+              "version_added": "25"
```

Comment:
This all looks great, just have one little question.  Chrome Desktop had the `webkit` prefix from version 5 to version 25.  Technically, wouldn't that mean Chrome Android 18 would have the `webkit` prefix as well?

Reply:
Fixed. Please look again.

Reply:
Thanks for clarification!